docs/reference/early-access.md

```diff
@@ -0,0 +1,56 @@
+---
+id: early-access
+title: Early access
+sidebar_label: Early access
+description: "Use early access releases to quickly learn about upcoming features and components."
+---
+
+Use early access releases to quickly learn about upcoming features and components.
+
+## Types of early access releases
+
+Selected Camunda features and components are released upfront as alpha or beta versions, e.g., to provide users the opportunity to test a feature and participate in the development by giving feedback prior to market launch (general availability, GA).
```

Comment:
It flows strangely with the ", e.g.,"
We should be able to clearly articulate why we're offering early access releases without having to say "example"

Reply:
Agreeing here